lib/utils/seedData.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable indent */
+const User = require('../models/User.js');
+const Budget = require('../models/Budget.js');
+const Expenditure = require('../models/Expenditure.js');
+
+const seedData = async() => {
```

Comment:
there is a lot of duplicate code here. You can make a function that contains the duplication. then call that function each time you need a user/budget/expendeture